### PR TITLE
update zh_CN.lang to #279

### PR DESCRIPTION
**Pull request by sfchipan**

### CODE REVIEW

File: src/main/resources/assets/powersuits/lang/zh_CN.lang [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#zh_CN
 Made by users: crafteverywhere, 3TUSK
 Last edited on 8/30/15 1:35 AM EST
```
Comment (3TUSK) [inline src/main/resources/assets/powersuits/lang/zh_CN.lang]: Probably you'd like to change the last edition time.

Comment (Joccob) [inline src/main/resources/assets/powersuits/lang/zh_CN.lang]: Line 127 动力全套 应该修正为 动力拳套

File: src/main/resources/assets/powersuits/lang/zh_CN.lang [hunk 2]
```diff
@@ -135,13 +140,29 @@ gui.install=安装
 gui.salvage=回收
 
 gui.newKeybind=新建
-gui.trashKeybind=删除!
+gui.trashKeybind=删除
 gui.pressKey=按键
 gui.keybindInstructions1=使用"新建"来绑定新快捷键.
 gui.keybindInstructions2=拖拽模块完成绑定.
 gui.keybindInstructions3=拖到"删除!"上解除绑定.
```
Comment (3TUSK) [inline src/main/resources/assets/powersuits/lang/zh_CN.lang]: line 143 has removed the exclamation mark; thus here should do the same thing.
(Using a computer which cannot type Chinese temporarily. I apologize for that.)

File: src/main/resources/assets/powersuits/lang/zh_CN.lang [hunk 3]
```diff
@@ -1,4 +1,3 @@
-#zh_CN
 Made by users: crafteverywhere, 3TUSK
```
Comment (3TUSK) [inline src/main/resources/assets/powersuits/lang/zh_CN.lang]: Actually... I suggest add your name into it.
--->`Made by users: crafteverywhere, 3TUSK, sf_chipan`

**Comment by 3TUSK** [comment]
Well... thanks for removing unnecessary slang usage.

**Comment by MachineMuse** [comment]
Sorry if this isn't sufficient; feel free to make additional pull requests to fix minor issues/typos/etc.

**Pull request merged**